coreservices/partsrelationshipservice/prs-parent/pom.xml

```
@@ -1,27 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>org.springframework.boot</groupId>
-        <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.5.4</version>
-        <relativePath/>
-        <!-- lookup parent from repository -->
+        <groupId>net.catenax.prs</groupId>
+        <artifactId>parts-relationship-service-root</artifactId>
+        <version>${revision}</version>
     </parent>
 
-    <groupId>com.catenax.prs</groupId>
+    <groupId>net.catenax.prs</groupId>
     <artifactId>prs-parent</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
     <packaging>pom</packaging>
 
     <name>PRS Parent</name>
-    <description>Parent module.</description>
+    <description>Parent module for PRS modules.</description>
 
     <properties>
         <java.version>11</java.version>
+        <maven.compiler.source>${java.version}</maven.compiler.source>
+        <maven.compiler.target>${java.version}</maven.compiler.target>
```

```suggestion
        <maven.compiler.source>${java.version}</maven.compiler.source>
        <maven.compiler.target>${java.version}</maven.compiler.target>
        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```
We can declare encoding here, Otherwise we have maven build warning:  `[WARNING] Using platform encoding (Cp1252 actually) to copy filtered resources, i.e. build is platform dependent!`